src/components/UI/StyledContainer.js

```diff
@@ -1,13 +1,25 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 const Container = styled.div`
-	background-color: white;
-	width: 384px;
-	height: 436px;
-	padding: 20px 0 20px 20px;
 	display: flex;
-	border-radius: 50px;
-	margin: 10px;
+
+	${({ variant }) =>
+		variant === 'product' &&
+		css`
+			background-color: white;
+			width: 90vw;
+			height: 436px;
+			padding: 20px 0 20px 20px;
+			border-radius: 50px;
+			margin: 10px;
+		`}
+	${({ variant }) =>
+		variant === 'imageUpload' &&
+		css`
+			width: 80%;
+			justify-content: space-around;
+			align-items: center;
+		`}
```

Comment:
I'd recommend to create two separate styled-components here. Your containers don't have anything in common expect for `display: flex`, so creating two separate files makes it cleaner.

I saw that you do the same in other styled-component-files as well. If the variants don't share any styling, they are not really variants but rather separate styled-components.

Reply:
Yes I guess I don't realy get the styled component idea after all. I will change it.